code/scripting/api/objs/control_binding.h

```diff
@@ -8,13 +8,20 @@
 namespace scripting {
 namespace api {
 
+/*
+ * A class wrapping the Control_Config's ActionID's to Lua. Lua can request a control-binding object by name and will get this class.
+ * This class, only holding the IoActionId as an identifier, can then be used to access and modify the control binding with this id.
+ */
 class cci_h {
  private:
-	IoActionId idx;
+	IoActionId idx; //!< The ActionId thos lua object references.
  public:
 	cci_h();
 	cci_h(int n_id);
 
+	/*
+	* @returns If this object holds a valid ActionId
```

Comment:
Style nitpick, "returns" tell what value is returned, here the grammar suggests that it doesn't return anything (or at all) if the ActionId is invalid.
```
/**
 * @returns true if this object holds a valid IoActionId, false otherwise
 */
 ```
 
Loaded values of bools, ints, and enums that have multiple meanings can opt to return the same value, but document each case of why that value is done.  Use your judgment as to whether its cleaner or easier to document them on a single line or multiple lines:
```
/**
 * @f haveEnoughGas()
 * @returns true If we have infinite gas, or
 * @returns true if we have enough gas to get to the target, or
 * @returns false otherwise
 */
 ```
```
 /**
  * @f haveEnoughGas()
  * @returns true if we have infinite gas or enough gas to get there, or
  * @returns false otherwise
  */
 ```